Demo/Demo/Models/Order.swift

```diff
@@ -1,6 +1,6 @@
 struct Order: Codable, Equatable {
 
-    enum CodingKeys: String, CodingKey {
+    enum OrderCodingKeys: String, CodingKey {
```

Comment:
It looks like everywhere else in the SDK we go with the default `CodingKeys`: https://github.com/search?q=repo%3Apaypal%2Fpaypal-ios%20CodingKeys&type=code. Is there a reason we want to diverge from that here?

Reply:
I just realized that we don't need any of these codingKeys. Just use 
 ```
let decoder = JSONDecoder()
decoder.keyDecodingStrategy = .convertFromSnakeCase
```
in parse function in` DemoMerchantAPI.` 
We already have keyEncodingStrategy in buildURLRequest function but not in parse function.

Reply:
Because of I get warning about nested logic if I have CodingKeys for Order and for Card. 
I just put decodingStrategy in parse method in DemoMerchantAPI so that there is no need for
CodingKeys for snakeCase. I realized that the CodingKey that I had defined for paymentMethodToken 
causes decoding error if I use this strategy on top of that, so I am removing the CodingKey there as well.

Reply:
CodingKeys taken out, replaced with decoding strategy in `parse` method of `DemoMerchantAPI` in commit e2f1c5967aa90e94f758493b70ef36e2732e78b3